src/layer/heatmap/IntervalType.js

```diff
@@ -0,0 +1,15 @@
+define([], function(){
+    "use strict";
+
+    /**
+     * Enumeration of different approaches towards the interval and color scales in case of heatmap.
+     * @exports IntervalType
+     * @type {{CONTINUOUS: number, QUANTILES: number}}
+     */
+    var IntervalType = {
+        CONTINUOUS: 0,
+        QUANTILES: 1
```

Comment:
Could you describe how these two options differ and why they should both be in this first version?

Reply:
@balhar-jakub, as discussed, please add comments about what these two options do.